src/style.css

```diff
@@ -1,3 +1,77 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font-family in case the main one doesn't work. For example, 'Roboto, Arial, sans-serif'.